testsuite/integration-arquillian/HOW-TO-RUN.md

```diff
@@ -475,38 +480,17 @@ To use a mobile browser you need to create a virtual device. The most convenient
 * **Supported mobile OS version:** iOS 11.x
 * **Run with:** `mvn clean test -Pios -Dappium.deviceName=device_name` where the device name is your device identification (e.g. `iPhone X`)
 
-## Run X.509 tests
```

Comment:
@slaskawi One last thing. We shouldn't completely remove the instruction for X.509 tests. They are still excluded by default (they cannot run on Undertow).
The only change should be removing {{auth.server.ssl.required}} property from the instructions.

Reply:
Of course I can do this but to be honest, after enabling TLS by default, the procedure is quite standard (just like for any other test):

```
mvn -f ./testsuite/integration-arquillian/pom.xml clean install -Pauth-server-wildfly -Dtest="org.keycloak.testsuite.hok.HoKTest"
```

So I believe it doesn't bring any new information to the table. That's why I decided to erase it.

Reply:
Ok, in this case I'm for removing:
https://github.com/keycloak/keycloak/blob/08a3f85e6da57de15945c0ae5aa7c9bf72d45206/testsuite/integration-arquillian/tests/base/pom.xml#L45
This is what makes the X.509 tests disabled no matter what. ;)

Reply:
It seems those tests are broken. Did we ever executed them (apart from manual execution)? If the answer is `no`, perhaps we could implement it in a separate JIRA?

Reply:
I tested them yesterday with your branch - no failures. :)
You need to execute them with `-Pauth-server-wildfly`. Without it it fails.

Reply:
Ok, so we reached conclusion. It turned out that x509 tests require some small adjustments. They can only run properly with PhantomJS browser (it supports self-signed certificates out of the box). 

I reverted relevant to x509 tests paragraphs from `HOW-TO-RUN.md` and created [KEYCLOAK-9512](https://issues.jboss.org/browse/KEYCLOAK-9512) to implement all required changes.